ci/packaging/suse/rpmfiles_maker.sh

```diff
@@ -41,6 +41,10 @@ then
       "$branch_project" "$branch_name"
     osc -A 'https://api.suse.de' co -o "$work_dir" \
       "$branch_project/$branch_name"
+    log "Running OBS services"
+    pushd "$work_dir"
+    osc -A 'https://api.suse.de' service disabledrun
+    popd
```

Comment:
In general, I dislike using pushd and popd around a command to run the command in a separate directory; I tend to favor the use of a subshell, as in
```bash
(
  cd "$workdir"
  command in dir
  some other command
)
```
That's portable to all POSIX shells, doesn't break everything after if the `popd` is forgotten (forgetting a paren will cause a helpful parse error), and the parens + indentation makes it a little more clear which commands are being "contained".

But it's also a personal preference.  This script explicitly uses `#!/bin/bash`, so bashisms are technically fine, and it's just one command, so there's little chanceof confusion. Just something to consider.